packages/solo/src/start.js

```diff
@@ -150,9 +150,11 @@ async function buildSwingset(
     }
     await initializeSwingset(config, argv, hostStorage);
   }
+  const slogFile = process.env.CLIENTSLOGFILE;
```

Comment:
In deference to #3581, since we get to decide what this variable is named, I'd suggest putting a subsystem-related prefix on the variable at the very least:

```suggestion
  const slogFile = process.env.SOLO_SLOGFILE;
```